stdlib/ballerina-builtin/src/main/ballerina/ballerina/file/natives.bal

```diff
@@ -2,16 +2,19 @@ package ballerina.file;
 
 import ballerina.io;
 
-@Description { value:"The Read access mode"}
-public const string R = "R";
-@Description { value:"The Write access mode"}
-public const string W = "W";
-@Description { value:"The Read Write access mode"}
-public const string RW = "RW";
-@Description { value:"The Append access mode"}
-public const string A = "A";
-@Description { value:"The Read Append access mode"}
-public const string RA = "RA";
+@Description { value:"The types of access modes available when dealing with files" }
+@Field { value:"R: Read-only access mode" }
+@Field { value:"W: Write-only access mode" }
+@Field { value:"RW: Read and write access mode" }
+@Field { value:"A: Append-only access mode" }
+@Field { value:"RA: Read and append access mode" }
```

Comment:
Shall we reformat properly- Extra space after the "{"

Reply:
Ack